Jenkinsfile

```diff
@@ -32,6 +29,7 @@ def startLisk() {
     cp test/config.json test/genesisBlock.json .
     export NODE_ENV=test
     BUILD_ID=dontKillMe ~/start_lisk.sh
+    sleep 10
```

Comment:
it shouldn't be the case. Before executing any tests we are running `waitUntilBlockchainReady` function which checks server availability